### PR TITLE
Update from MAGFest master

### PR DESCRIPTION
**Pull request by kitsuta**
Fixes https://github.com/MidwestFurryFandom/mff-rams-plugin/issues/60, for real this time. Fixes https://github.com/MidwestFurryFandom/mff-rams-plugin/issues/86.

We don't necessarily need the other changes, but it doesn't hurt to stay up-to-date where possible.

### CODE REVIEW

File: uber/model_checks.py [hunk 1]
```diff
@@ -235,6 +235,12 @@ def promo_code_is_useful(attendee):
     if attendee.promo_code:
```
Comment (kitsuta) [inline uber/model_checks.py]: We override this in he event plugin, so changes here won't affect us.

File: uber/site_sections/registration.py [hunk 2]
```diff
@@ -22,8 +22,14 @@
 
 
 def pre_checkin_check(attendee, group):
-    if c.NUMBERED_BADGES and not attendee.badge_num:
-        return 'Badge number is required'
+    if c.NUMBERED_BADGES:
```
Comment (kitsuta) [inline uber/site_sections/registration.py]: No major changes here, code got moved from several lines down to fix a bug for events without numbered badges.

File: uber/templates/base.html [hunk 3]
```diff
@@ -125,6 +125,38 @@
           }
         }
     </style>
+
+    {% block feedback_javascript %}
```
Comment (kitsuta) [inline uber/templates/base.html]: ... so uh, @RobRuana, is feeding any event's instance of RAMS into MAGFest's JIRA intended behavior?

Comment (jmdawson) [reply uber/templates/base.html]: I feel like we want an answer to this before we pull this in.

**Comment by RobRuana** [comment]
MAGFest _is_ cool with bug reports from other events! If we receive any bug reports that are MFF-specific, I'll forward them on to you 😄

**Comment by kitsuta** [comment]
@jmdawson Done -- see `fix-mff60-mff86` for a branch where MAGFest master is merged into mff-staging.

**Pull request merged**